packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,54 @@
+import express, { Router } from 'express';
+import { Contribution, ContributionConstructorValues } from '../entities/Contribution';
+import logger from '../logger';
+
+export const contributions = Router();
+
+contributions.get('', async (req, res) => {
+    if(req.user)
+    {
+        try {
+            /*
+            * Plan: 
```

Comment:
Would it make sense / be worth it to store the node id of the user in the contribution table (in the cron job pr) to make this easier? If that happened, then the plan would just be to entityManager.find all contribution entries with the current user's node id and return that.